### ISSUE TITLE
Is it possible to use LoRa models and custom models for this? 

### ISSUE THREAD

**Issue by hourlyray1709** [bes-dev/stable_diffusion.openvino#153]
I know --model param exists, but I am not sure how to use it. I tried putting the path to the model for another stable diffusion app I'm using, but it didn't work. 

I also wish to use codeformers to fix faces, so is it possible to add it to the code to fix faces? Thanks

**Comment by phipgn** [comment]
same question here, someone please help to answer

**Comment by CCMKCCMK** [comment]
same question

**Comment by RedAndr** [comment]
No, current code doesn't have such an option. Although you can merge a LoRA into a model.

**Comment by Drake53** [comment]
If you want these kind of features you should just use something like https://github.com/AUTOMATIC1111/stable-diffusion-webui
It's about 2~3 times slower without openvino but to compensate you get way more features, and you can generate at lower resolutions like 384x384 if you want to speed things up a bit.
I haven't tried it but there's also an openvino fork: https://github.com/openvinotoolkit/stable-diffusion-webui
